iml-gui/crate/src/page/mod.rs

```diff
@@ -72,6 +72,36 @@ pub(crate) enum Page {
     Volume(volume::Model),
 }
 
+impl Page {
+    pub(crate) fn title(self: &Self) -> String {
+        match self {
+            Self::About => "About".into(),
+            Self::AppLoading => "Loading...".into(),
+            Self::Filesystems(_) => "Filesystems".into(),
+            Self::Filesystem(m) => format!("Filesystem: {}", &m.fs.name),
+            Self::Dashboard(_) => "Dashboard".into(),
+            Self::FsDashboard(m) => format!("{} Filesystem Dashboard", &m.fs_name),
+            Self::ServerDashboard(m) => format!("{} Server Dashboard", &m.host_name),
+            Self::TargetDashboard(m) => format!("{} Target Dashboard", &m.target_name),
+            Self::Jobstats => "Jobstats".into(),
+            Self::Login(_) => "Login".into(),
+            Self::Mgts(_) => "MGTs".into(),
+            Self::NotFound => "Page not found".into(),
+            Self::OstPools => "OST Pools".into(),
+            Self::OstPool(m) => format!("OST Pool: {}", &m.id),
```

Comment:
```suggestion
            Self::OstPool(m) => format!("OST Pool: {}", &m.name),
```

Reply:
It is not ready yet :) Like other places with ids.

Reply:
Still wouldn't hurt to add it now. We'll need to remember to update this when we implement the page otherwise.

Reply:
Well, I have just a rough idea that those models will contain respective entities. I could add them, but then add code to initialize them, etc. And all it just to fix this small issue instead of working on those features for real. On the other side, I am sure the first thing to do will be removal the dummy ids from those model which will break this code remind to update it.